cmd/policy-tool/debug/debug.go

```diff
@@ -0,0 +1,64 @@
+package debug
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/forseti-security/config-validator/pkg/gcv"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "debug",
+	Short:   "Run the config validator on a set of policies / cai data and print out any info on errors.",
+	Example: `policy-tool debug --policies ./forseti-security/policy-library/policies --libs ./forseti-security/policy-library/libs`,
+	RunE:    lintCmd,
+}
+
+var (
+	flags struct {
+		policies []string
+		libs     string
+		files    []string
+	}
+)
+
+func init() {
+	Cmd.Flags().StringSliceVar(&flags.policies, "policies", nil, "Path to one or more policies directories.")
+	Cmd.Flags().StringVar(&flags.libs, "libs", "", "Path to the libs directory.")
+	Cmd.Flags().StringSliceVar(&flags.files, "file", nil, "Files to process.")
+	if err := Cmd.MarkFlagRequired("policies"); err != nil {
+		panic(err)
+	}
+}
+
+func lintCmd(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
func debugCmd(cmd *cobra.Command, args []string) error {
```

Reply:
oof, the embarrassment of not completely fixing a cut and paste job, fixed.